Description: Escaping the "#" character looks like not needed anymore and now causing build failures
because the autogenerated c file contains an extra \ at the begin
Author: Gianfranco Costamagna <locutusofborg@debian.org>

--- gst-plugins-good1.0-1.16.2.orig/common/glib-gen.mak
+++ gst-plugins-good1.0-1.16.2/common/glib-gen.mak
@@ -5,7 +5,7 @@
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_enum_prefix=gst_color_balance
 
-enum_headers=$(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers=$(foreach h,$(glib_enum_headers),\n#include \"$(h)\")
 
 # these are all the rules generating the relevant files
 %-marshal.h: %-marshal.list
--- gst-plugins-good1.0-1.16.2.orig/common/gst-glib-gen.mak
+++ gst-plugins-good1.0-1.16.2/common/gst-glib-gen.mak
@@ -6,9 +6,9 @@
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=#include <gst/foo/foo-prelude.h>
 
-enum_headers=$(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers=$(foreach h,$(glib_enum_headers),\n#include \"$(h)\")
 
 # these are all the rules generating the relevant files
 $(glib_gen_basename)-marshal.h: $(glib_gen_basename)-marshal.list
